### ISSUE TITLE
Perplexity value after pre-training?

### ISSUE THREAD

**Issue by leannmlindsey** [Zhihan1996/DNABERT_2#46]
This was not reported in the paper but I was interested in what the perplexity value was after pre-training.  Thank you,
LeAnn

**Comment by leannmlindsey** [comment]
I am also interested in how many sequences you used to train the DNA tokenizer and what sequences those were

**Comment by Zhihan1996** [comment]
Hey, the mask token prediction accuracy is about 30 percents and the MLM loss on the evaluation set is about 4.31 after the training. Also, the data used for tokenizer training is the entire pre-training corpus we used.
